extensions/variants.go

```diff
@@ -218,6 +222,20 @@ func parseFuncName(compoundName string) (name string, args ArgumentList) {
 	return name, args
 }
 
+func minArgumentCount(paramTypeList ArgumentList, variadicBehavior *VariadicBehavior) int {
+	if variadicBehavior == nil {
+		return len(paramTypeList)
```

Comment:
your test calls `MinArgumentCount` but somehow this entire function isn't covered by the tests, how is that?

Reply:
I think this test coverage script doesn't detect coverage hit across pacakge. I have tried it in IDE and it shows fine.
I will figure out and fix as a separate PR

<img width="1345" alt="Screenshot 2024-09-25 at 12 46 51 PM" src="https://github.com/user-attachments/assets/1ce04b5e-93c5-405c-b7be-88e64a732c1a">